receiver/jaegerreceiver/config_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaegerreceiver
+
+import (
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/census-instrumentation/opencensus-service/internal/configmodels"
+	"github.com/census-instrumentation/opencensus-service/internal/configv2"
+	"github.com/census-instrumentation/opencensus-service/internal/factories"
+)
+
+var _ = configv2.RegisterTestFactories()
+
+func TestLoadConfig(t *testing.T) {
+	factory := factories.GetReceiverFactory(typeStr)
+
+	config, err := configv2.LoadConfigFile(t, path.Join(".", "testdata", "config.yaml"))
+
+	require.NoError(t, err)
+	require.NotNil(t, config)
+
+	assert.Equal(t, len(config.Receivers), 2)
+
+	r0 := config.Receivers["jaeger"]
+	assert.Equal(t, r0, factory.CreateDefaultConfig())
+
+	r1 := config.Receivers["jaeger/customname"].(*ConfigV2)
+	assert.Equal(t, r1,
+		&ConfigV2{
+			Protocols: map[string]*configmodels.ReceiverSettings{
+				"thrift-http": {
+					Enabled:  false,
+					Endpoint: "example.com:3456",
```

Comment:
nit: this is just a test but the endpoint doesn't make much sense, it should be a local address or empty to indicate all available network interfaces.

Reply:
Yeah, for the purpose of the particular test it doesn't matter. I can change to avoid confusion.